src/js/app.js

```diff
@@ -0,0 +1,67 @@
+$(document).ready(function() {
+
+
+
+
+$('#searchBtn').on('click', function(event) {
+  event.preventDefault();
+
+
+  $.get('https://www.reddit.com/search.json', {
+
+    q: searchInput,
+  }).done(function(data) {
+    intoArray(data);
+    // console.log(data);
+  }).fail(function(data) {
+    console.log('failed!');
+  });
+
+
+});
+//function to grab the input from the text box and put it into the q field above
+function searchInput() {
+  return $('#input').val();
+}
+
+function intoArray(data) {
```

Comment:
> There are two hard things in computer science: cache invalidation, naming things, and off-by-one errors.

You're guilty of the second one. What is `intoArray`? An alternative name for this function may be... `processImages`

Same goes with `doStuff`. An alternative name for that function may be... `startSlideshow`